### PR TITLE
[MIXEDARCH-275] Creating a cluster with multi-architecture compute machine on IBM Power

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.14

Issue: https://issues.redhat.com/browse/MIXEDARCH-275

Draft: https://docs.google.com/document/d/1ZIc3Zdag2wI9qWKrrildJJyh5Wqfhmkndz11lRGNZZI/edit#heading=h.ewr4uspzt7p7

Link to docs preview: https://64357--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc
Comment (SNiemann15) [inline post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:9]:
```suggestion
To create a cluster with multi-architecture compute machines on {ibmpowerProductName}(`ppc64le`), you must have an existing single-architecture bare metal (`x86_64`) cluster. For more information on bare metal installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metalInstalling a user provisioned cluster on bare meta]. You can then add `ppc64le` compute machines to your {product-title} cluster.
```

Comment (SNiemann15) [reply post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:9]: Reverting my suggestion, after discussion with SMEs for IBM Z we hae removed the "bare metal" wording in my [PR](https://github.com/openshift/openshift-docs/pull/63855)

File: post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc
Comment (prb112) [inline post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:14]: Should we mention any gotcha's here? Or is that better for a technote? 

Otherwise LGTM

**Review by prb112 — APPROVED** [review]
/lgtm

File: post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc
Comment (alishaIBM) [reply post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:14]: Thanks @prb112 
Updated the PR with a note. Request you to check. Thanks.

File: modules/machine-user-infra-machines-pxe.adoc
Comment (prb112) [inline modules/machine-user-infra-machines-pxe.adoc:51]: are lines 47 to 49 suppose to use ibm-power since it state x86_64 and aarch64, i think it should have a different ifndef

**Review by prb112 — APPROVED** [review]
LGTM, i answered my prior comment

File: modules/machine-user-infra-machines-pxe.adoc
Comment (alishaIBM) [inline modules/machine-user-infra-machines-pxe.adoc:89]: @prb112 
I have replaced `aarch64` with `ppc64le`. Request you to review if the steps hold good for `ppc64le`.

Comment (alishaIBM) [reply modules/machine-user-infra-machines-pxe.adoc:89]: Preview link: https://64357--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power#machine-user-infra-machines-pxe_creating-multi-arch-compute-nodes-ibm-power

**Review by prb112 — APPROVED** [review]
LGTM

File: modules/machine-user-infra-machines-pxe.adoc
Comment (prb112) [reply modules/machine-user-infra-machines-pxe.adoc:89]: lgtm
/ack

File: post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc
Comment (manojnkumar) [reply post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:14]: @alishaIBM :  Can you remove the reference to KVM?

Comment (alishaIBM) [reply post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:14]: @manojnkumar I have removed the reference to KVM. 
Is it okay to write: "The following procedures explain how to create a RHCOS compute machine using an ISO image or network PXE booting."
Please confirm. Thanks.

Comment (prb112) [reply post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:14]: That's fine.

**Review by prb112 — APPROVED** [review]

**Comment by manojnkumar** [comment]
> @manojnkumar can I go ahead with the peer review ?

Yes.

**Comment by manojnkumar** [comment]
LGTM

**Review by ShaunaDiaz — APPROVED** [review]
Good work on a complex topic! Just a few thoughts and questions for your consideration.

File: post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc
Comment (alishaIBM) [reply post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc:13]: Agree. Added.

**Comment by alishaIBM** [comment]
@ShaunaDiaz thanks for the comments. I have updated the PR. Request you to please check. Thanks.

File: modules/multi-architecture-verifying-cluster-compatibility.adoc
Comment (stevsmit) [inline modules/multi-architecture-verifying-cluster-compatibility.adoc:29]: I'm going to suggest something that the team has talked about, but that there is no written rule for (other than saying not to "overuse admonitions." https://redhat-documentation.github.io/supplementary-style-guide/#admonitions. So consider this change, but I won't hold up another merge-review because of it. 

In the past, we've tried to stay away from stacking admonitions in favor of having 1 admonition with 2 bullet points. Because there's no rule on this I usually wouldn't hold it up for merge review, but "please" needs to be removed in this note as well, as per IBMSG: I'm going to suggest something that the team has talked about, but that there is no written rule for (other than saying not to "overuse admonitions." https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

In the past, we've tried to stay away from stacking admonitions in favor of having 1 admonition with 2 bullet points. Because there's no rule on this I usually wouldn't hold it up for merge review, but "please" needs to be removed in this note as well: 

```suggestion
When using multiple architectures, hosts for the {product-title} nodes must share the same storage layer. If they do not have the same storage layer, use a storage provider such as `nfs-provisioner`.
```

So, I'm requesting the removal of please, and the consideration of using bullet points in one NOTE instead of stacking 2 notes.

File: modules/multi-architecture-verifying-cluster-compatibility.adoc
Comment (stevsmit) [inline modules/multi-architecture-verifying-cluster-compatibility.adoc:65]: Do you want to add an Additional Resources under this module in the assembly so that users don't have to go searching for Migrating to a cluster with multi-architecture compute machines? 

I do see that it is x-reff'd in an IMPORTANT admonition in the following section, though, so up to you.

**Comment by stevsmit** [comment]
A few suggestions before merging.

**Comment by alishaIBM** [comment]
@stevsmit thank you for the suggestions. I have updated the PR. Request you to check. Thanks.

File: modules/multi-architecture-verifying-cluster-compatibility.adoc
Comment (opayne1) [inline modules/multi-architecture-verifying-cluster-compatibility.adoc:34]: The use of "one" seems strange to me. Is this referring to the user? I would replace with "you" if so.

Comment (opayne1) [reply modules/multi-architecture-verifying-cluster-compatibility.adoc:34]: I dont see anything against it in the style guides, but it seems it would be clearer if you used "you".

File: modules/multi-architecture-verifying-cluster-compatibility.adoc
Comment (opayne1) [inline modules/multi-architecture-verifying-cluster-compatibility.adoc:29]:
```suggestion
When using multiple architectures, hosts for {product-title} nodes must share the same storage layer. If they do not have the same storage layer, use a storage provider such as `nfs-provisioner`.
```

File: modules/multi-architecture-verifying-cluster-compatibility.adoc
Comment (opayne1) [inline modules/multi-architecture-verifying-cluster-compatibility.adoc:68]:
```suggestion
To migrate your cluster so the cluster supports multi-architecture compute machines, follow the procedure in xref:../../updating/updating_a_cluster/migrating-to-multi-payload.adoc#migrating-to-multi-payload[Migrating to a cluster with multi-architecture compute machines].
```

**Review by opayne1 — COMMENTED** [review]
So sorry to catch a few more small things to consider before merging. Ping me when you are ready and I will merge for you!

File: modules/multi-architecture-verifying-cluster-compatibility.adoc
Comment (alishaIBM) [reply modules/multi-architecture-verifying-cluster-compatibility.adoc:34]: Agree. Updated the PR.

**Comment by alishaIBM** [comment]
@opayne1 I have resolved the comments. Request you to check.

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**
